sonic_!amd64.go

```diff
@@ -0,0 +1,36 @@
+//go:build !amd64
+
+/*
+ * Copyright 2022 ByteDance Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sonic
+
+import (
+	"reflect"
```

Comment:
please use the same format with other sonic codes:
- single quote import package
- four space as indent

Reply:
Okay.
I was wondering why we do not use `gofmt` to format golang code.

Is there a script tool or shell script to automatically format it?